src/services/F2fService.ts

```diff
@@ -119,6 +120,24 @@ export class F2fService {
 		}
 	}
 
+	async sendToGovNotify(event: GovNotifyEvent): Promise<void> {
+		const messageBody = JSON.stringify(event);
+		console.log('messageBody1', messageBody);
+		const params = {
+			MessageBody: messageBody,
+			QueueUrl: process.env.GOV_NOTIFY_QUEUE_URL,
```

Comment:
We seem to be adding more and more to environment variables not sure I understand why.. GOVUKNOTIFY_API_KEY_SSM_PATH is added to environment variables and as such every lambda needs that env variable just to import EnvironmentVariables.ts even if it's not required by that lambda (Just an example)

Reply:
Agree that lambdas should only have the environment variables they need to function. It keeps them decoupled from one another. In general we should avoid sharing code between lambdas that we wouldn't extract out to be a library at some point, so if `EnvironmentVariables.ts` is a util file containing per lambda methods I can see an argument for keeping it but we're still introducing [connasence of Identity](https://connascence.io/identity.html) when we don't need to be. This change as it is meets the AWS Serverless best practices and the internal GDS best practices listed in confluence [here](https://govukverify.atlassian.net/wiki/spaces/DISBP/pages/3060662308/Best+Practices).

Reply:
I think all of these lambdas are required for F2F CRI flow and would be sharing some code thereby. The ask here is to keep env vars checks in one place for maintainability. The env var as its being used in the code above has no null check and instead of adding it every time its being used, best to keep them in a method.